### ISSUE TITLE
pip install: AddModelFromFile fails due to missing `drake/package.xml`

### ISSUE THREAD

**Issue by EricCousineau-TRI** [RobotLocomotion/drake#16069]
Was trying to write up example (inline def. of SDFormat file, parsing it, etc.), but found the following error (min. repro for Ubuntu 18.04, CPython 3.6.9):

```sh
$ python3 -m venv .
$ source ./bin/activate
$ pip install -U pip wheel
$ pip install drake==0.35.0

$ python3 <<EOF
from pydrake.all import MultibodyPlant, Parser
Parser(MultibodyPlant(0.0)).AddModelFromFile("/this/file/does/not/exist")
EOF

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: Could not find Drake resource_path 'drake/package.xml' because Drake CMake install
marker specified a resource root of '{venv}/lib/python3.6/site-packages/pydrake/lib/../share'
but that root did not contain the expected file
'{venv}/lib/python3.6/site-packages/pydrake/lib/../share/drake/package.xml'.

$ find . -name 'package.xml'
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/iiwa_description/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/wsg_50_description/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/realsense2_description/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/jaco_description/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/ycb/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/allegro_hand_description/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/manipulation/models/franka_description/package.xml
./lib/python3.6/site-packages/pydrake/share/drake/examples/manipulation_station/models/package.xml
```

**Comment by jwnimmer-tri** [comment]
Oh.  The `package.xml` file isn't even in the wheel.  That would do it!  The wheel is also missing the `geometry/**` meshcat files.  I think the build-wheel script should default to copying all shared resources, with a denylist for the stuff we don't want (resource_tool, repo.bzl, install_prereqs), rather than trying to keep an allowlist up to date.

**Comment by jwnimmer-tri** [comment]
(High priority, since I think nobody can use sdformat nor urdf files at all without this.)

**Comment by jwnimmer-tri** [comment]
For now, the work-around for users would be to copy that [package.xml](https://github.com/RobotLocomotion/drake/blob/v0.35.0/package.xml) file to `python3.6/site-packages/pydrake/share/drake/package.xml` by hand.

@mwoehlke-kitware if you have time today to PR a fix here, that would be helpful.  Otherwise, I'll get to it at the end of the day.

**Comment by jwnimmer-tri** [comment]
This fix was released in `drake==0.36.0`.
